conversant/prompt_chatbot.py

```diff
@@ -426,14 +429,15 @@ def configure_client(self, client_config: Dict = {}) -> None:
         if not hasattr(self, "client_config"):
             self.client_config = {
                 "model": "xlarge",
-                "max_tokens": 100,
+                "max_tokens": 200,
                 "temperature": 0.75,
                 "frequency_penalty": 0.0,
                 "presence_penalty": 0.0,
-                "stop_sequences": ["\n"],
+                "stop_sequences": ["\nUser:", self.bot_name],
             }
         # Override default config values with the config passed in
         if isinstance(client_config, Dict):
+            client_config["stop_sequences"] = ["\nUser:", "\n{}".format(self.bot_name)]
```

Comment:
I think this line breaks the functionality of passing in custom stop sequences as a `client_config`, since it assigns it to the `client_config["stop_sequences"]`, overriding any user defined stop sequences. Wondering what's the purpose of this line!

Reply:
Here's a test that shows this: in [this line](https://github.com/cohere-ai/sandbox-conversant-lib/blob/5e78c1667db42aed3fa90dc76d78443c345b8677/tests/test_prompt_chatbot.py#L67), add
```python
    assert prompt_chatbot.client_config["stop_sequences"] == ["\n"]
```